dvc/utils/serialize/_common.py

```diff
@@ -42,13 +43,23 @@ def __init__(self, path: "AnyPath", message: str):
         from dvc.utils import relpath
 
         path = relpath(path)
+        self.path = path  # For programmatic consumption, e.g. Studio.
```

Comment:
I don't think this comment is useful here.

Reply:
Added it to explain why the field is there while it isn't used in DVC itself.

But that info will also be in the git history + we explicitly test that the field is there, so I'm also fine with removing it!